### ISSUE TITLE
Setup automerge for dependabot PRs

### ISSUE THREAD

**Issue by joshunrau** [DouglasNeuroInformatics/DouglasDataCapturePlatform#102]
Once build workflow is done, implement the following action:

1. Trigger on PR from dependabot
2. Trigger the CI pipeline
3. If the build passes and the dependency is not a major update by NPM convention, then merge into main
